openmoc/plotter.py

```diff
@@ -1798,13 +1798,13 @@ def _get_pixel_coords(plot_params):
                    root.getMaxZ() - TINY_MOVE]
 
     if not plot_params.xlim is None:
-        bounds['x'] = plot_params.xlim
+        bounds['x'] = list(plot_params.xlim)
```

Comment:
xlim ylim zlim is originally tuple of 2 floats.

Reply:
This is a fix for Python2. In Python 3 this all works without turning the tuple in a list